java/ql/lib/semmle/code/java/Javadoc.qll

```diff
@@ -48,7 +52,7 @@ class Javadoc extends JavadocParent, @javadoc {
     if isEolComment(this)
     then result = ""
     else (
-      if strictcount(this.getAChild()) = 1 then result = " */" else result = " ... */"
+      if count(this.getAChild()) <= 1 then result = " */" else result = " ... */"
```

Comment:
If you reverse the condition the you can keep `strictcount`, i.e. this is simpler and does the same thing:
```suggestion
      if strictcount(this.getAChild()) > 1 then result = " ... */" else result = " */"
```